RadioItem/tests/RadioItem-specs.js

```diff
@@ -1,19 +1,61 @@
-import {shallow} from 'enzyme';
+import '@testing-library/jest-dom';
+import {fireEvent, render} from '@testing-library/react';
 
-import {RadioItemBase} from '../RadioItem';
+import RadioItem, {RadioItemBase} from '../RadioItem';
 
 describe('RadioItem Specs', () => {
-	test('should support a custom icon', () => {
-		const expected = 'check';
 
-		const subject = shallow(
-			<RadioItemBase icon={expected}>
-				Hello RadioItem
-			</RadioItemBase>
-		);
+	test('should support a custom icon', () => {
+		const {getByRole} = render(<RadioItemBase icon="check" selected>Hello RadioItem</RadioItemBase>);
+		const radioItemElement = getByRole('checkbox');
 
-		const actual = subject.find('.icon').prop('children');
+		const expected = '✓';
+		const actual = radioItemElement.children.item(1).textContent;
 
 		expect(actual).toBe(expected);
 	});
+
+	test('should render `children`', () => {
+		const {getByText} = render(<RadioItemBase>Hello RadioItem</RadioItemBase>);
+
+		const actual = getByText('Hello RadioItem');
+
+		expect(actual).toBeInTheDocument();
+	});
+
+	test('should render a disabled RadioItem when `disabled` is true', () => {
+		const {getByRole} = render(<RadioItemBase disabled>Hello RadioItem</RadioItemBase>);
+
+		expect(getByRole('checkbox', {disabled: true}));
```

Comment:
interesting approach :)

Reply:
it seems to be a false positive expectation. if I put "disabled:false", the test still passes

Reply:
You're right, I thought it would behave like Checkbox. Thanks for suggestion